internal/devbox/generate/tmpl/prod.Dockerfile.tmpl

```diff
@@ -0,0 +1,28 @@
+FROM jetpackio/devbox:latest
+
+WORKDIR /code
+USER root:root
+RUN mkdir -p /code && chown ${DEVBOX_USER}:${DEVBOX_USER} /code
+USER ${DEVBOX_USER}:${DEVBOX_USER}
+
+{{- /*
+Ideally, we first copy over devbox.json and devbox.lock and run `devbox install` 
+to create a cache layer for the dependencies. This is complicated because
+devbox.json may include local dependencies (flakes and plugins). We could try
+to copy those in (the way the dev Dockerfile does) but that's brittle because
+those dependencies may also pull in other local dependencies and so on. Another
+sulution would be to add a new flag `devbox install --skip-errors` that would 
+just try to install what it can, and ignore the rest.
+
+A hack to make this simpler is to install from the lockfile instead of the json.
+*/}}
+
+COPY --chown=${DEVBOX_USER}:${DEVBOX_USER} . .
+
+RUN devbox install
```

Comment:
There was an issue with using `devbox install` command in dockerfile. IIRC it didn't activate the python plugin so the subsequent python commands were not running in a virtual environment.

Reply:
this is the issue I'm talking about:
https://github.com/jetify-com/devbox/issues/1122

can you test the steps in the issue to make sure we don't regress?

Reply:
I'm a bit confused by that issue. I understand that `install` doesn't run init hooks, but I would assume the line below

`RUN devbox shellenv --init-hook >> ~/.profile`

Would do the initialization.

Furthermore, actually activating the virtual environment is a manual process so neither install/run would do it.

Do you know why shellenv is not working?

Reply:
FWIW, since the prod dockerfile always ends in `devbox run start` it will always run the init hook which creates the virtenv. To activate it, the user would need to add that to their start command. Maybe we can add an env variable that automatically activates it.